config/routes.yaml

```diff
@@ -18,6 +18,10 @@ privacy_policy:
     path:     /privacy-policy
     defaults: { _controller: JoliCode\SecretSanta\Controller\ContentController::privacyPolicy }
 
+faq:
+    path:     /faq
+    defaults: { _controller: JoliCode\SecretSanta\Controller\ContentController::faq }
```

Comment:
You can use TemplateController
https://symfony.com/doc/current/templating/render_without_controller.html